src/main/java/server/RouteFactory.java

```diff
@@ -0,0 +1,29 @@
+package server;
+
+import java.util.ArrayList;
+
+public class RouteFactory implements IRouteFactory {
+
+    public ArrayList<Route> makeRoutes() {
+        return new ArrayList<Route>() {{
+            add(new Route(Method.GET, "/simple_get", (request) -> {
+                return new ResponseBuilder().build();
+            }));
+            add(new Route(Method.HEAD, "/get_with_body", (request) -> {
+                return new ResponseBuilder().build();
+            }));
+            add(new Route(Method.GET, "/method_options", (request) -> {
+                return new ResponseBuilder().build();
+            }));
+            add(new Route(Method.GET, "/method_options2", (request) -> {
+                return new ResponseBuilder().build();
+            }));
+            add(new Route(Method.PUT, "/method_options2", (request) -> {
+                return new ResponseBuilder().build();
+            }));
+            add(new Route(Method.POST, "/method_options2", (request) -> {
+                return new ResponseBuilder().build();
+            }));
+        }};
+    }
+}
```

Comment:
Something that might be interesting to think about would be how you could extract the hard coded routes from the routes factory. Maybe something like a config file where you can specify the routes you want for your server? Think of something like a rails routes.rb file